packages/components/src/components/SquiggleViewer/ExpressionViewer.tsx

```diff
@@ -18,16 +18,17 @@ import { NumericFunctionChart } from "../FunctionChart/NumericFunctionChart.js";
 import { ScatterChart } from "../ScatterChart/index.js";
 import { generateDistributionPlotSettings } from "../PlaygroundSettings.js";
 import { ItemSettingsMenu } from "./ItemSettingsMenu.js";
-import { VariableBox } from "./VariableBox.js";
+import { SQTypeWithCount, VariableBox } from "./VariableBox.js";
```

Comment:
We use `Sq` instead of `SQ` everywhere else.

Reply:
I thought that this was a bit different because it's just a component, but happy to change.